cmd/server/flags.go

```diff
@@ -544,4 +544,9 @@ var flags = []cli.Flag{
 		Name:    "encryption-disable-flag",
 		Usage:   "Flag to decrypt all encrypted data and disable encryption on server",
 	},
+	&cli.BoolFlag{
```

Comment:
please document this and add a warning site, that it's not recommended to enable it